src/Files.App/ViewModels/UserControls/StatusCenterViewModel.cs

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.App.ViewModels
+{
+	public class OngoingTasksViewModel : ObservableObject, IOngoingTasksActions
+	{
+		public ObservableCollection<StatusBanner> StatusBannersSource { get; private set; } = new ObservableCollection<StatusBanner>();
+
+		private int medianOperationProgressValue = 0;
+		public int MedianOperationProgressValue
+		{
+			get => medianOperationProgressValue;
+			private set => SetProperty(ref medianOperationProgressValue, value);
+		}
+
+		public int OngoingOperationsCount
+		{
+			get
+			{
+				int count = 0;
+
+				foreach (var item in StatusBannersSource)
+				{
+					if (item.IsProgressing)
+					{
+						count++;
+					}
+				}
+
+				return count;
+			}
+		}
+
+		public bool AnyOperationsOngoing
+		{
+			get => OngoingOperationsCount > 0;
+		}
```

Comment:
Can't we use a simple lambda?
```suggestion
		public bool AnyOperationsOngoing
			=> OngoingOperationsCount > 0;
```